examples/riot-kitchen-sink/src/stories/addon-backgrounds.stories.js

```diff
@@ -0,0 +1,27 @@
+import { storiesOf } from '@storybook/riot';
+import { withBackgrounds } from '@storybook/addon-backgrounds';
+import ButtonRaw from 'raw-loader!./Button.tag';
```

Comment:
If loading .tag as raw, we should add it to the default webpack config for riot

Reply:
The tag can be loaded as a component or as a string,
depends on developer's experience.
Some of them will instantiate the component with "mount" (with the riot-tag-loader)
And some of them will prefer to instantiate using a jsx-like notation (therefore with the raw-loader).